<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>build_run</class>
 <widget class="QWidget" name="build_run">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>675</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QFrame" name="model_status_frame">
           <property name="styleSheet">
            <string notr="true">
	background-color: rgb(208, 208, 208);
	border-radius: 25px;

	color: black;


</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>15</number>
            </property>
            <item row="0" column="0" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Optimization Model Solve Status:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTextEdit" name="report_progress_box">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius: 10px;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
	background-color: rgb(208, 208, 208);
	border-radius: 25px;

	color: black;


</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>30</number>
         </property>
         <item row="5" column="4">
          <widget class="QPushButton" name="run_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
        border: 1px solid rgb(40, 84, 113);
        border-radius: 7px;
        background-color: rgb(40, 84, 113);
        color: white;
		 font: 700 10pt &quot;Segoe UI&quot;;
        padding: 1px;
    }
QPushButton:hover {
        background-color: rgb(60, 120, 150);
    }
QPushButton:pressed {
        background-color: rgb(20, 60, 80);
    }</string>
           </property>
           <property name="text">
            <string>Solve</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Select Solver:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>12</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Select Results Folder:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QToolButton" name="build_run_help_button">
           <property name="toolTipDuration">
            <number>3</number>
           </property>
           <property name="styleSheet">
            <string notr="true">
    QToolButton:hover {
        background-color: rgb(230, 230, 230);
		border-radius:5px;
    }
    QToolButton:pressed {
        background-color: rgb(180, 180, 180);		
		 border-radius:5px;
    }</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../resources.qrc">
             <normaloff>:/icon/images/icons/ainfo_24dp_5F6368_FILL0_wght200_GRAD0_opsz24.png</normaloff>:/icon/images/icons/ainfo_24dp_5F6368_FILL0_wght200_GRAD0_opsz24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QComboBox" name="solver_select_box">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <item>
            <property name="text">
             <string>Gurobi</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>GLPK</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Clp</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>HiGHs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Other (Upcoming)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QComboBox" name="results_file_box">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="browse_folder_button">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
        border: 1px solid rgb(40, 84, 113);
        border-radius: 7px;
        background-color: rgb(40, 84, 113);
        color: white;
		 font: 700 10pt &quot;Segoe UI&quot;;
        padding: 1px;
    }
QPushButton:hover {
        background-color: rgb(60, 120, 150);
    }
QPushButton:pressed {
        background-color: rgb(20, 60, 80);
    }</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="build_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
        border: 1px solid rgb(40, 84, 113);
        border-radius: 7px;
        background-color: rgb(40, 84, 113);
        color: white;
		 font: 700 10pt &quot;Segoe UI&quot;;
        padding: 1px;
    }
QPushButton:hover {
        background-color: rgb(60, 120, 150);
    }
QPushButton:pressed {
        background-color: rgb(20, 60, 80);
    }</string>
           </property>
           <property name="text">
            <string>Build</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="styleSheet">
         <string notr="true">QPushButton {
        border: 1px solid rgb(40, 84, 113);
        border-radius: 7px;
        background-color: rgb(40, 84, 113);
        color: white;
		 font: 700 10pt &quot;Segoe UI&quot;;
        padding: 1px;
    }
QPushButton:hover {
        background-color: rgb(60, 120, 150);
    }
QPushButton:pressed {
        background-color: rgb(20, 60, 80);
    }</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QPushButton" name="previous_button">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Previous</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">image: url(:/logos/images/logo/optimization_icon.png);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="next_button">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
